fw/http_msg.c

```diff
@@ -1550,11 +1550,18 @@ tfw_h2_msg_cutoff_headers(TfwHttpResp *resp, TfwHttpRespCleanup* cleanup)
 
 			if ((begin <= off) && (end >= off)) {
 				it->frag = -1;
-				/* TODO: Handle this case, most likely for low MTU
-				 * it will be implemented as part of #1703 */
-				return;
+				/* We would end up here if the start of the body or
+				 * the end of CRLF lies within the linear data area
+				 * of the current @it->skb
+				 */
+				ret = ss_skb_linear_transform(it->skb_head,
+							      it->skb, body);
```

Comment:
why we use body as a split point not "off"?

Reply:
In case, when there is no body, we don't really want to split anything. We just zap the linear part and pretend that this SKB is kind of empty and start adding the fragments.